packages/docusaurus-search-local/src/server/index.ts

```diff
@@ -158,6 +160,7 @@ export default function cmfcmfDocusaurusSearchLocal(
       titleBoost,
       contentBoost,
       parentCategoriesBoost,
+      ignore,
     },
   } = options;
 
```

Comment:
Could you add a check here that checks that every entry 
- starts with a slash
- does not end with a slash (while still allowing an entry that is just `/`)

and throw an error otherwise?